.storybook/deprecated/splitbutton/splitbutton.yml

```diff
@@ -1,6 +1,6 @@
 name: Split button
 status: Deprecated
-deprecationNotice: Use a <a href="buttongroup.html">button group</a> to show any additional actions related to the most critical action. Reference <a href="https://spectrum.corp.adobe.com/page/button-group/#Use-a-button-group-to-show-additional-actions">Spectrum documentation</a> for more information.
+deprecationNotice: Use a [button group](buttongroup.html) to show any additional actions related to the most critical action. Reference [Spectrum documentation](https://spectrum.corp.adobe.com/page/button-group/#Use-a-button-group-to-show-additional-actions) for more information.
```

Comment:
```suggestion
deprecationNotice: Use a <a class="spectrum-Link" href="buttongroup.html">button group</a> to show any additional actions related to the most critical action. Reference <a class="spectrum-Link" href="https://spectrum.corp.adobe.com/page/button-group/#Use-a-button-group-to-show-additional-actions">Spectrum documentation</a> for more information.
```

Reply:
I found that using markdown instead of HTML here automatically handles dark mode, are you seeing something different in the [PR deploy](https://pr-2527--spectrum-css.netlify.app/splitbutton)?

Reply:
Oh!  Didn't even check - just based it off what I just saw in the other PR. Good to go!